astronomer/airflow/version_check/update_checks.py

```diff
@@ -232,8 +232,38 @@ def __init__(self):
 
     def available_update(self):
         from .models import AstronomerAvailableVersion
+        END_OF_LIFE = {'1.10.12': pendulum.datetime(2022,1,1), '1.10.14': pendulum.datetime(2022,1,1),
```

Comment:
I wonder... should we add this to updates.astronomer.io instead? By baking it into the image, we can't change it down the line otherwise.

Reply:
I thought about that too. I think the dates can be changed due to some circumstances, so unless it won't change then baking it is fine

Reply:
Makes sense

Reply:
Not sure how to add it there

Reply:
Yeah, let’s restrict the scope to only showing the version from the current release series.

The platform team is handling the logic separately in https://github.com/astronomer/issues/issues/3018 to mark certain release as deprecated.